src/containers/BookList.js

```diff
@@ -1,24 +1,27 @@
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
-import { createBookAction, removeBookAction } from '../actions';
 
 const BookList = ({ books }) => {
   const bookRows = books.map(book => (<Book key={`book-number-${book.bookId}`} book={book} />));
-
   return (
     <div>
       <table>
```

Comment:
As stated in the specific requirement above this has to also connect to removeBook.
`
const mapDispatchToProps = dispatch => ({
  removeBook: id => dispatch(removeBook(id))
}
`
> Note the `handleRemoveBook` relies on the remove book action to remove a book from the store.

Reply:
I did that already